### DISCUSSION TITLE
Welcome to Play Framework Discussions!

### DISCUSSION THREAD

**Discussion by mkurz** [playframework/playframework#11212] [Announcements]
## 👋 Welcome!

We’re using Discussions as a place to connect with other members of our community. We hope that you:
* Ask questions you’re wondering about.
* Share ideas.
* Engage with other community members.
* Welcome others and are open-minded. Remember that this is a community we build together 💪.

You can still access our previous (read only) discussion forums:
  * https://discuss.lightbend.com/c/play/8
  * https://groups.google.com/g/play-framework

**Comment by reid-spencer** [comment]
Q: What is the disposition of the discourse server given that GitHub Discussions are being used now?

**Reply by BillyAutrey** [reply]
We're trying to narrow down to GitHub and Discord for all conversations.  Everything else is "read-only", for search/historical purposes.
